packages/react/src/runtime/shorthand.ts

```diff
@@ -0,0 +1,105 @@
+import type {
+  Depths,
+  ShorthandProperties,
+  shorthandBuckets as ShorthandBucketsType,
```

Comment:
Sanity check: this import doesn't bloat bundle sizes, right?  We have that protection, so CI would fail if this increased from 500bytes => 4kb again or whatnot.  `type` imports should do this, just making sure this "magic" works as I expect.

Reply:
the `import type ...` statements don't get included in `dist/` yeah